packages/use-contractkit/src/constants.tsx

```diff
@@ -53,6 +55,15 @@ export const PROVIDERS: {
     showInList: () => true,
     listPriority: () => 0,
   },
+  [SupportedProviders.CeloDance]: {
```

Comment:
Please move this down below Metamask. Additions added later should not be given high placement in the list, sorry!